shared/src/test/diff/nu/UserDefinedAnnotations.mls

```diff
@@ -59,3 +59,16 @@ module dummy
 //│ ║  l.56: 	@helloa
 //│ ╙──      	 ^^^^^^
 //│ module dummy
+
+// TODO: In the future, we should still support annotations that require a constructor.
+:e
```

Comment:
Comments starting with "TODO" are interpreted specially (a wart we should remove).
```suggestion
:e // TODO: In the future, we should still support annotations that require a constructor.
```

Reply:
In this case, is keeping it as a TODO fine?